addons/core/functions/fnc_vehicleId.sqf

```diff
@@ -19,7 +19,9 @@
 */
 params ["_unit"];
 
-if (isNull (objectParent _unit) && isNil {_unit getVariable "TFAR_vehicleIDOverride"}) exitWith {"no"};//Unit is not in vehicle
+private _vehicleOverrideEnabled = !isNil {_unit getVariable "TFAR_vehicleIDOverride"};
```

Comment:
this is the hot path.
The normal case is that its not enabled and the function immediately exits returning "no"

you invert the value with `!` twice in the hotpath, if you remove the ! on both you save two command calls